### PR TITLE
Added Wishlist and Cart Page

### PR DESCRIPTION
**Pull request by vrinda-mahajan**
Created the wishlist and cart page. Please review my PR.
Cart Preview: https://eager-yonath-6d3921.netlify.app/pages/cart/cart
Wishlist Preview: https://eager-yonath-6d3921.netlify.app/pages/wishlist/wishlist

### CODE REVIEW

File: Pages/ProductPage/ProductPage.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
     top: 8rem;
     overflow-y: scroll;
```
Comment (Maianki) [inline Pages/ProductPage/ProductPage.css:7]: Suggestion: You can use ```overflow-y: auto``` so that scroll is only visible when content overflows

**Review by Maianki — APPROVED** [review]
Clean code and nice UI. Good to merge.

**Comment by hsnice16** [comment]
UI is awesome, clean, right color choice. The code looks good too. Nice work.

**Review by hsnice16 — APPROVED** [review]

File: Pages/ProductPage/ProductPage.css (hunk 1 above)
Comment (vrinda-mahajan) [reply Pages/ProductPage/ProductPage.css:7]: Thanks for the suggestion.

**Review by foolhardy21 — APPROVED** [review]
looks amazing. suggestion - in the cart page you can make it as flex column for smaller screens.

**Pull request merged**